test/binCases/ARGUMENTS.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
We can do that. Wanted to leverage a easier module loading syntax to catch that data.